docs/atlascli/command/atlas-customDns-aws-describe.txt

```diff
@@ -77,3 +77,10 @@ If the command succeeds, the CLI returns output similar to the following sample.
    <Enabled>
    
 
+Examples
+--------
+
+.. code-block::
+
+   # Return the details for the custom DNS configuration deployed to AWS in the project with ID 618d48e05277a606ed2496fe:		
+   atlas customDns aws describe  --projectId 618d48e05277a606ed2496fe 
```

Comment:
[optional] There are 2 spaces between the command and  --projectId, is it intentional?

Reply:
Ah yes, sorry about that. I just fixed it :) @tibulca, would you mind LGTM-ing again?